src/main.rs

```diff
@@ -459,31 +527,38 @@ impl<'a> ResolvedPackage<'a> {
                     cwd
                 )
             )?;
-            writeln!(f, "features = builtins.concatLists [")?;
-            for (feature, optionality) in self.features.iter() {
-                let mut f = f.indent(2);
-                match optionality
-                    .to_expr(outer.root_features, n_root_pkgs)
-                    .simplify()
-                {
-                    True => writeln!(f, "[ {:?} ]", feature)?,
-                    expr => writeln!(f, "({} ({}) {:?})", outer.optional, expr.to_nix(), feature)?,
+            if self.features.len() != 0 {
```

Comment:
```suggestion
            if !self.features.is_empty() {
```

Reply:
ah. I tried `empty()`.. did not it is `is_*` form.